locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1446 @@
+msgid "Delete post"
+msgstr "Supprimer le message"
+
+#: bookwyrm/templates/snippets/status/status_options.html:23
+#: bookwyrm/templates/snippets/user_options.html:13
+msgid "Send direct message"
+msgstr "Envoyer un message direct"
+
+#: bookwyrm/templates/snippets/switch_edition_button.html:5
+msgid "Switch to this edition"
+msgstr "Changer vers cette édition"
+
+#: bookwyrm/templates/snippets/tag.html:14
+msgid "Remove tag"
+msgstr "Supprimer le tag"
+
+#: bookwyrm/templates/tag.html:7
+#, python-format
+msgid "Books tagged \"%(tag.name)s\""
+msgstr "Livres tagués « %(tag.name)s »"
+
+#: bookwyrm/templates/user/create_shelf_form.html:5
+msgid "Create New Shelf"
+msgstr "Créer une nouvelle étagère"
+
+#: bookwyrm/templates/user/create_shelf_form.html:22
+msgid "Create shelf"
+msgstr "Créer l’étagère"
+
+#: bookwyrm/templates/user/edit_shelf_form.html:5
+msgid "Edit Shelf"
+msgstr "Modifier l’étagère"
+
+#: bookwyrm/templates/user/edit_shelf_form.html:26
+msgid "Update shelf"
+msgstr "Mettre l’étagère à jour"
+
+#: bookwyrm/templates/user/followers.html:30
+#, python-format
+msgid "%(username)s has no followers"
+msgstr "%(username)s n’a pas d’abonné(e)"
+
+#: bookwyrm/templates/user/following.html:17
+msgid "Following"
+msgstr "Abonné(e) à"
+
+#: bookwyrm/templates/user/following.html:30
+#, python-format
+msgid "%(username)s isn't following any users"
+msgstr "%(username)s n’est abonné(e) à personne"
+
+#: bookwyrm/templates/user/lists.html:28
+msgid "Create list"
+msgstr "Créer une liste"
+
+#: bookwyrm/templates/user/user.html:7
+msgid "User profile"
+msgstr "Profil"
+
+#: bookwyrm/templates/user/user.html:13
+msgid "Edit profile"
+msgstr "Modifier le profil"
+
+#: bookwyrm/templates/user/user.html:24
+#: bookwyrm/templates/user/user_layout.html:66
+msgid "Shelves"
+msgstr "Étagères"
+
+#: bookwyrm/templates/user/user.html:29
+#, python-format
+msgid "See all %(size)s"
+msgstr "Voir les %(size)s"
+
+#: bookwyrm/templates/user/user.html:42
+#, python-format
+msgid "See all %(shelf_count)s shelves"
+msgstr "Voir les %(shelf_count)s étagères"
+
+#: bookwyrm/templates/user/user.html:54
+#, python-format
+msgid "Set a reading goal for %(year)s"
+msgstr "Définir un défi lecture pour %(year)s"
+
+#: bookwyrm/templates/user/user.html:60
+msgid "User Activity"
+msgstr "Activité du compte"
+
+#: bookwyrm/templates/user/user.html:63
+msgid "RSS feed"
+msgstr "Flux RSS"
+
+#: bookwyrm/templates/user/user.html:74
+msgid "No activities yet!"
+msgstr "Aucune activité pour l’instant !"
+
+#: bookwyrm/templates/user/user_layout.html:30
+msgid "Follow Requests"
+msgstr "Demandes d’abonnement"
+
+#: bookwyrm/templates/user/user_layout.html:48
+msgid "Activity"
+msgstr "Activité"
+
+#: bookwyrm/templates/user/user_layout.html:54
+msgid "Reading Goal"
+msgstr "Défi lecture"
+
+#: bookwyrm/templates/user/user_preview.html:13
+#, python-format
+msgid "Joined %(date)s"
+msgstr "Enregistré(e) il y a %(date)s"
```

Comment:
it looks like the built in django date formatter is adding `il y a` on its own, so I think just `"Enregistré(e) %(date)s"` will do the trick here?

Reply:
This is fixed in https://github.com/mouse-reeve/bookwyrm/pull/665/commits/4a178d05cda54c70bac7f46a877bbbe1ab8262da.